plugin/kite.vim

```diff
@@ -22,7 +22,7 @@ filetype on
 
 
 if !exists('g:kite_supported_languages')
-  let g:kite_supported_languages = ['python']
+  let g:kite_supported_languages = ['python', 'javascript']
```

Comment:
Ideally the plugin would query Kite's enabled languages? Is this already implemented in the plugin?
Python is always enabled, but JS or Golang could still be turned off at this time.